namespaces/cloud-platform-live-0.k8s.integration.dsd.io/support-labelling-app-dev/namespace.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: support-labelling-app-dev 
+  labels:
+    name: support-labelling-app-dev 
```

Comment:
I'm not sure if we've decided we're going forward with [this](https://github.com/ministryofjustice/cloud-platform-environments/pull/109) - should we add the labels & annotations anyway?